packages/fast-components-react-base/src/tabs/tabs.props.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { IManagedClasses, ITabsClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export enum Orientation {
```

Comment:
This is a pretty non-specific name so I think it should either be:
a.) Semantically scoped to Tabs (eg, `TabsOrientation`)
b.) Made generic in a non-tabs specific location